go.mod

```diff
@@ -6,10 +6,13 @@ require (
 	github.com/btcsuite/btcutil v1.0.2 // indirect
 	github.com/calebcase/tmpfile v1.0.2 // indirect
 	github.com/gogo/protobuf v1.3.1 // indirect
+	github.com/oschwald/geoip2-golang v1.4.0
 	github.com/oschwald/maxminddb-golang v1.7.0
 	github.com/spacemonkeygo/monkit/v3 v3.0.7-0.20200515175308-072401d8c752
 	github.com/spf13/cobra v1.0.0
+	github.com/storj/storjmap v0.0.0-20200616005132-b4224cbbe795
```

Comment:
You should not use this: `github.com/storj/storjmap`

Reply:
done